cfme/common/provider_views.py

```diff
@@ -249,7 +249,10 @@ class ProviderTemplatesView(ProviderVmsTemplatesView):
 
     @property
     def is_displayed(self):
-        title = '{name} (All Miq Templates)'.format(name=self.context['object'].name)
+        if self.extra.appliance.version < '5.10':
```

Comment:
First use I've seen of the extra attribute!

Reply:
This looks like a good place to use `VersionPick`, since it is within a view method, and should pick itself when needed.